Doc/library/random.rst

```diff
@@ -303,6 +297,15 @@ be found in any statistics text.
 Alternative Generator
 ---------------------
 
+.. class:: Random([seed])
+
+   Class to be subclassed if you want to use a different basic generator of your
```

Comment:
This wording suggests to me that the class is only useful as a base class for subclassing, whereas I think it's also useful to instantiate and use directly.